pyglmnet/datasets.py

```diff
@@ -138,29 +170,33 @@ def fetch_rgc_data(dpath, accept_rgcs_license=False):
     dpath : str
         The data path
     """
-    if accept_rgcs_license:
-        answer = 'y'
-    else:
-        answer = input('%s\nAgree (y/[n])? ' % _rgcs_license_text)
-    if answer.lower() != 'y':
-        raise RuntimeError('You must agree to the license to use this '
-                           'dataset')
+    dpath = get_data_home(data_home=dpath)
```

Comment:
while you're at it, could you also fix this for the other fetchers?

Reply:
Working on it